verifier/src/qe_identity.rs

```diff
@@ -197,14 +199,13 @@ impl Tcb {
 /// signature.
 #[derive(Debug, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
-pub struct SignedQeIdentity<'a> {
-    #[serde(borrow)]
-    enclave_identity: &'a RawValue,
+pub struct SignedQeIdentity {
+    enclave_identity: Box<RawValue>,
```

Comment:
I originally wanted this to be an owned RawValue so that it would be more portable, but punted at the time.  To make the tests more manageable I finally confronted the hurdle I was facing.
I think in general the JSON string is small enough that copying here is acceptable, especially with the added ergonomics of being able to move this around without persisting a `&str` somewhere.